remi/server.py

```diff
@@ -360,18 +360,20 @@ def _instance(self):
             head.add_child('internal_css', "<link href='/res:style.css' rel='stylesheet' />\n")
             
             body = gui.BODY()
+            body.add_class('remi-main')
             body.onload.connect(self.onload)
             body.onerror.connect(self.onerror)
             body.ononline.connect(self.ononline)
             body.onpagehide.connect(self.onpagehide)
             body.onpageshow.connect(self.onpageshow)
             body.onresize.connect(self.onresize)
+
             self.page = gui.HTML()
             self.page.add_child('head', head)
             self.page.add_child('body', body)
 
             if not hasattr(self, 'websockets'):
-                self.websockets = []
+                self.websockets = set()
```

Comment:
@smurfix Why you preferred to use a *set*? Just curiosity

Reply:
Basically because it has a `.discard` method. I got a crash when I closed two views at the almost-same time and a set allowed me to fix that problem without making the code more complicated. Note that there's another `.websockets.remove` call which I intentionally didn't replace with `discard`.

A set also is more semantically valid (lists in general have an assumed ordering and might contain duplicates) and is more efficient when more people share a view.

Reply:
Good solution, thank you for the explanation